### ISSUE TITLE
how to set extraQueryParameters for login_hint?

### ISSUE THREAD

**Issue by phoenix377** [rmcfarlane82/react-native-msal-plugin#2]
I tried to do set login_hint as follow:

`const extraQueryParameters = 'login_hint=xxx@outlook.com';`
`await authClient.aquireTokenB2CAsync(AzureConfig.SCOPES, AzureConfig.POLICIES, extraQueryParameters)`

But it adds login_hint in url like following:
`https://...&%22login_hint=partnertest2018%40outlook.com%22`
And it's not adding email field correctly.

How can I set login_hint using extraQueryParameters for aquireTokenB2CAsync function?

**Comment by rmcfarlane82** [comment]
hi, extra query params are used like below.

``` js
const extraQueryParams = { someParameterName: "parameterValue", anotherParameter: "anotherValue" };

await authClient.aquireTokenB2CAsync(AzureConfig.SCOPES, AzureConfig.POLICIES, extraQueryParams)

```

However when passing { login_hint:"user@test.com" } the parameter is removed so it looks like you can't pass it into extra query parameters.

There is another overload that takes loginhint as a parameter in the msal library, i'll add another issue to implement it for both platforms.

**Comment by rmcfarlane82** [comment]
this is resolved in release 2.0.0, there is now a login hint parameter on acquireTokenAsync method
